### ISSUE TITLE
Need more validations for a large number of users

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/sidewalk-quality-analysis#57]
I just posted this to the main Sidewalk repo: https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2139 but, in short, there is a significant number of users who completed one mission but have yet to receive 10 validations.

Hmm, I need to check how many of those users actually supplied 10 labels... let me check on that.

**Comment by jonfroehlich** [comment]
The updated stats on this are:

> 77/1512 users (5.1%) never completed even one audit mission. Removing them from analysis.
0/1313 users (0.0%) have not provided the min num of labels of 10. Removing them from analysis.
0/1313 users (0.0%) have not received the min num of validations of 10. Removing them from analysis.
We filtered out 199/1512 users (13.16%)

Mikey, do you prefilter users out who have not yet received the minimum number of 10 validations in your data dumps to me?

**Comment by jonfroehlich** [comment]
Something fishy is going on here because 1512-77 is not 1313... stay tuned.

**Comment by misaugstad** [comment]
Agreed. There's too much code being written in SQL that should be happening in Python. I can remove the filtering from my end and generate new datasets for you today if you'd like.

**Comment by misaugstad** [comment]
I'm about to upload new CSVs with the min validations requirement removed. I'll let you close this issue once you've made any changes that you want to make on the Python notebook side.
